src/components/Onboarding.js

```diff
@@ -1,43 +1,30 @@
 import { h } from 'preact'
-import { useSoftkey, useI18n, useRange } from 'hooks'
+import { useSoftkey, useI18n } from 'hooks'
 import { onboarding, goto } from 'utils'
 
 export const Onboarding = () => {
   const i18n = useI18n()
-  const [currentIndex, prevOnboard, nextOnboard] = useRange(0, 3)
 
   const exitOnboard = () => {
     onboarding.markAsDone()
     goto.search()
   }
 
-  const getImageBackgroundStyle = index => {
-    // only the first onboard image doesn't have background image
-    return index ? { backgroundImage: `url(/images/onboarding-${index}-background.png)` } : {}
-  }
-
-  const softkeyConfig = [
-    { left: i18n('softkey-skip'), onKeyLeft: exitOnboard, right: i18n('softkey-next'), onKeyRight: nextOnboard, onKeyArrowRight: nextOnboard },
-    { left: i18n('softkey-back'), onKeyLeft: prevOnboard, onKeyArrowLeft: prevOnboard, right: i18n('softkey-next'), onKeyRight: nextOnboard, onKeyArrowRight: nextOnboard },
-    { left: i18n('softkey-back'), onKeyLeft: prevOnboard, onKeyArrowLeft: prevOnboard, center: i18n('softkey-get-started'), onKeyCenter: exitOnboard }
-  ]
-  useSoftkey('onboarding', softkeyConfig[currentIndex], [currentIndex], true)
+  useSoftkey('Onboarding', {
+    center: i18n('softkey-get-started'),
+    onKeyCenter: exitOnboard
+  })
 
   return (
     <div class='onboarding'>
-      <div class='image' style={getImageBackgroundStyle(currentIndex)}>
-        <img src={`/images/onboarding-${currentIndex}.png`} />
+      <div class='image'>
+        <img src='/images/onboarding-0.png' />
       </div>
       <div class='title'>
-        {i18n(`onboarding-${currentIndex}-title`)}
+        {i18n('onboarding-0-title')}
       </div>
       <div class='description'>
-        {i18n(`onboarding-${currentIndex}-description`)}
-      </div>
-      <div class='indicator'>
-        <div class={`dot ${currentIndex === 0 ? 'selected' : ''}`} />
-        <div class={`dot ${currentIndex === 1 ? 'selected' : ''}`} />
-        <div class={`dot ${currentIndex === 2 ? 'selected' : ''}`} />
+        {i18n('onboarding-0-description')}
```

Comment:
Considered renaming the `onboarding-0` to just `onboarding` but `AboutWikipedia` uses the same keys so opted to leave as is